product-service/lambda-handlers/getAllProducts.js

```diff
@@ -1,9 +1,13 @@
-import products from '../products.json';
 import ProductsService from '../services/products.service';
 
 export const getAllProducts = async event => {
   try{
-    const productsServiceInstance = new ProductsService( products );
+    console.info( "getAllProducts handler was invoked" );
```

Comment:
not sure, but you could log event object here

Reply:
because, for example, FE could pass some session-id header and filtering by this session-id could be applied if logging system supports it of course

Reply:
I added changes in commit:
https://github.com/Major98113/aws-shop-be/pull/2/commits/cf8099d47975b71430f4f62bccaf92720afe73f5